rdmd.d

```diff
@@ -27,13 +27,15 @@ version (Posix)
     enum objExt = ".o";
     enum binExt = "";
     enum libExt = ".a";
+    enum dllExt = ".so";
```

Comment:
this should be `.dylib` when built on Mac, otherwise looks good

Reply:
Good catch, fixed.